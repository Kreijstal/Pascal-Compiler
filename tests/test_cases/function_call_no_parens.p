program FunctionCallNoParens;

uses SysUtils;

procedure Foo;
begin
end;

function Bar: integer;
begin
  Bar := 1;
end;

function GetFortyTwo: integer;
begin
  GetFortyTwo := 42;
end;

begin
  Foo;
  writeln(Bar);
  writeln(GetFortyTwo);
  writeln('Sum: ', Bar + GetFortyTwo);
end.
