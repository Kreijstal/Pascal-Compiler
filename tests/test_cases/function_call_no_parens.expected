1
42
Sum: 43
